crates/bevy_animation/src/lib.rs

```diff
@@ -625,7 +625,9 @@ fn apply_animation(
             let Some(target) = entity_from_path(root, path, children, names, cached_path) else {
                 continue;
             };
+
             any_path_found = true;
+
```

Comment:
```suggestion
            any_path_found = true;
```
why adding those newlines?

Reply:
added by accident